e2e/README.md

```diff
@@ -14,10 +14,14 @@ curl -sSfL https://raw.githubusercontent.com/longhorn/longhorn/master/scripts/en
 
 ### Run the test
 
-1. Deploy all backupstore servers (including `NFS` server and `Minio` as s3 server) for test purposes.
+1. Deploy all backupstore servers (including `NFS` server and `Minio` as s3 server, `CIFS` and `Azurtize` server) for test purposes.
+
+   For Azurtize, there are some manual steps need to be done after manifest deployed(https://confluence.suse.com/display/LON/Setup+Azurite+Backupstore+For+Testing).
```

Comment:
Could we have a test report for the CIFS and Azurite robot test cases run on our Jenkins?

Reply:
Test case "Test Incremental Restore" passed on my jenkins with `cifs` and `azurite`, because our jenkins need to point to my branch to run the scripts, currently it can not have report on our jenkins.